packages/@aws-cdk/cloud-assembly-schema/.gitignore

```diff
@@ -1,5 +1,6 @@
 lib/*.js
 test/*.js
+scripts/*.js
```

Comment:
Why not *.js?

Reply:
Left over from when the script was js and not ts.

Reply:
👍

Reply:
Actually this way its aligned to the other packages. Leaving it as is.